budgetportal/openspending.py

```diff
@@ -159,12 +159,18 @@ def get_subprogramme_dimension(self):
     def get_department_name_ref(self):
         return self.get_ref(self.get_department_dimension(), 'label')
 
+    def get_government_ref(self):
+        return self.get_ref(self.get_government_dimension(), 'key')
+
     def get_vote_number_ref(self):
         return self.get_ref(self.get_department_dimension(), 'key')
 
     def get_department_dimension(self):
         return self.get_dimension('administrative_classification')
 
+    def get_government_dimension(self):
+        return self.get_dimension('administrative_classification', level=1)
```

Comment:
this isn't right - government should be geo which exists - I think when you uploaded provincial budget vs actual you set government to administrative when it should be geo source and now when you query for it you have to use administration.

administration is used for department

Reply:
ah okay, makes sense. I wasn't sure which dimension fit best. In this case department is top level and gov is level 1.